tests/platform_helper/test_command_codebase.py

```diff
@@ -30,169 +26,49 @@ def mock_aws_client(get_aws_session_or_abort):
 
 
 class TestCodebasePrepare:
-    @patch("dbt_platform_helper.commands.codebase.requests.get")
-    def test_codebase_prepare_generates_the_expected_files(self, mocked_requests_get, tmp_path):
-        from dbt_platform_helper.commands.codebase import prepare
-
-        mocked_response_content = """
-            builders:
-              - name: paketobuildpacks/builder-jammy-full
-                versions:
-                  - version: 0.3.294
-                  - version: 0.3.288
-              - name: paketobuildpacks/builder-jammy-base
-                versions:
-                  - version: 0.1.234
-                  - version: 0.5.678
-              - name: paketobuildpacks/builder
-                deprecated: true
-                versions:
-                  - version: 0.2.443-full
-        """
-
-        def mocked_response():
-            r = requests.Response()
-            r.status_code = 200
-            type(r).content = PropertyMock(return_value=mocked_response_content.encode("utf-8"))
-
-            return r
-
-        mocked_requests_get.return_value = mocked_response()
-
-        os.chdir(tmp_path)
-
-        subprocess.run(["git", "init"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        subprocess.run(
-            ["git", "remote", "add", "origin", "git@github.com:uktrade/test-app.git"],
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
-        )
+    @patch("dbt_platform_helper.commands.codebase.Codebase")
+    def test_codebase_prepare_calls_codebase_prepare_method(self, mock_codebase_object):
+        mock_codebase_object_instance = mock_codebase_object.return_value
 
         result = CliRunner().invoke(prepare)
```

Comment:
I will no longer be confused by this once the codebase command method has been renamed.

```suggestion
        result = CliRunner().invoke(codebase)
```

It looked to me like we were asserting that the method we just invoked has been called.